buildtest/menu/build.py

```diff
@@ -111,6 +149,35 @@ def func_build_subcmd(args):
     # Discover list of one or more config files based on path provided
     config_files = discover_configs(args.config)
 
+    # if no files discovered let's stop now
+    if not config_files:
+        msg = "There are no config files to process."
+        logger.error(msg)
+        sys.exit(msg)
+
+    print("\n")
+    print("{:^45}".format("Discovered Files"))
+    print("\n")
+    for config in config_files:
+        print(config)
+    print("\n")
+    print("\n")
+
+    logger.debug(
+        f"Based on input argument: -c {args.config} buildtest discovered the following configuration {config_files}"
+    )
+
+    if args.exclude:
+        logger.debug(f"The exclude config pattern are the following: -e {args.exclude}")
+
+    config_files = [
+        config for config in config_files if include_file(config, args.exclude)
+    ]
+
+    logger.debug(
```

Comment:
What do you mean normalized?

Reply:
Bad naming, what about *filtered*, need some way to denote that configs discovered after exclusion are the ones that buildtest will process.